src/main/kotlin/com/beust/klaxon/Render.kt

```diff
@@ -10,7 +10,17 @@ private fun <A: Appendable> A.renderString(s: String): A {
             '\\' -> append(ch).append(ch)
             '\n' -> append("\\n")
             '\r' -> append("\\r")
-            else -> append(ch)
+            else -> {
+                if(isPrintableUnicode(ch)){
```

Comment:
Yes, we won't. 
I will add more test data and cases to show it.
RFC and json.org defines it:
https://tools.ietf.org/html/rfc7159#page-8